src/Chapter2.hs

```diff
@@ -336,7 +348,7 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b l = if a < 0 then [] else take (b - a + 1) (drop a l)
```

Comment:
Nice!
There is one more case to check, if `b < 0`. But actually, it is enough just to check if `b < a` and if it is to return the empty list 👍🏼 This will emit unnecessary `take`-`drop` manipulations in case of not good input values 🙂